### PR TITLE
rearranged to accommodate all possible database states

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: modules/ROOT/pages/database-administration/standard-databases/errors.adoc [hunk 1]
```diff
@@ -60,56 +60,27 @@ A database management operation may fail for a number of reasons.
...
+* `unknown`
 
 Most often, when a database management operation fails, Neo4j attempts to transition the database in question to the `offline` state.
 If the system is certain that no store files have yet been created, it transitions the database to `initial` instead.
 Similarly, if the system suspects that the store files underlying the database are invalid (incomplete, partially deleted, or corrupt), then it transitions the database to `dirty`.
 
 [NOTE]
 ====
-While `dropped` is a valid database state, it is only transiently observable, as database records are removed from `SHOW DATABASE` results once the `DROP` operation is complete.
+Additionally, `dropped` is also a valid database state, but it is only transiently observable, as database records are removed from `SHOW DATABASE` results once the `DROP` operation is complete.
```
Comment (gfx54b) [inline modules/ROOT/pages/database-administration/standard-databases/errors.adoc:83]: Not even, that, today in 5.x there is no code-path which could show you `dropped`/

**Review by gfx54b — COMMENTED** [review]
One small comment otherwise good to go.

File: modules/ROOT/pages/database-administration/standard-databases/errors.adoc (hunk 1 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/database-administration/standard-databases/errors.adoc:83]: `dropped` is now dropped 👍

File: modules/ROOT/pages/database-administration/standard-databases/errors.adoc [hunk 2]
```diff
@@ -60,57 +60,24 @@ A database management operation may fail for a number of reasons.
 For example, if the file system instance has incorrect permissions, or Neo4j itself is misconfigured.
 As a result, the contents of the `statusMessage` column in the `SHOW DATABASE` query results may vary significantly.
 
-However, databases may only be in one of a select number of states:
+However, databases may only be in one of a select number of states (see xref::database-administration/standard-databases/manage-databases.adoc#database-states[Database states] for more information):
```
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/errors.adoc:63]: Maybe we could add the link after the states' list so we don't interrupt the reading flow.

File: modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc [hunk 3]
```diff
@@ -182,13 +187,61 @@ However, some privileges enable users to see additional databases regardless of
 
 If a user has not been granted `ACCESS` privilege to any databases nor any of the above special cases, the command can still be executed but it will only return the `system` database, which is always visible.
 
+[[database-states]]
+=== Database states
+
+A database's `currentStatus` can be one of the following.
```
Comment (renetapopova) [inline modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc:193]:
```suggestion
A database's `currentStatus` can be one of the following:
```

**Review by renetapopova — APPROVED** [review]
Thanks, @AlexicaWright! I have two minor comments.

File: modules/ROOT/pages/database-administration/standard-databases/errors.adoc (hunk 2 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/database-administration/standard-databases/errors.adoc]: Yes, I agree, that's better!

File: modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc [hunk 4]
```diff
@@ -182,13 +187,61 @@ However, some privileges enable users to see additional databases regardless of
...
+| This state implies an error has occurred.
+The database's underlying store files may be invalid.
+For more information, consult the `statusMessage` column or the server's logs.
+
+| quarantined
+| The database is effectively stopped and its state may not be changed until no longer quarantined.
+For more information, consult the `statusMessage` column or the server's logs.
+
+| unknown
+| This instance of Neo4j doesn’t know the state of this database.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc:235]:
```suggestion
| This instance of Neo4j does not know the state of this database.
```

**Review by JPryce-Aklundh — APPROVED** [review]
Looks good 👍

**Pull request merged**